client/src/router/Config.ts

```diff
@@ -32,11 +34,30 @@ const RouterConfig: RouterConfigType[] = [
     component: VectorSearch,
     auth: true,
   },
-  {
-    path: '/system',
-    component: SystemView,
-    auth: true,
-  },
 ];
 
+function importAll(r: any, outOfRoot = false) {
+  r.keys().forEach((key: any) => {
+    const content = r(key);
+    const dirName = key.split('/config.json').shift().split('/')[1];
+    const pathName = content.client?.path;
+    const entry = content.client?.entry;
+    if (!(pathName || entry)) return;
```

Comment:
it's looks like we both need pathName and entry ?

Reply:
Has renamed `entry` to `fileEntry`.
`pathName` for route path.
`fileEntry` for load component from target file